app/controllers/cinema_halls_controller.rb

```diff
@@ -1,4 +1,76 @@
 # frozen_string_literal: true
 
 class CinemaHallsController < ApplicationController
+	def index 
+		@cinema_halls = Cinema_hall.all
+
+		respond_to do |format|
+			format.html  
+			format.json  { render :json => @cinema_halls }
+		end
+	end
+
+	def new
```

Comment:
Actions like `new` are mostly related to Rails Views. I see that your application generally is not created in [api-only mode](https://guides.rubyonrails.org/api_app.html) - it's not a problem, but try to implement only those functions which are valid for API mode - `index`, `show`, `update`, `create`, `destroy`. Also remember that we want to create only JSON API (at least for now) so try to compact your actions only to result:
```
render :json => @cinema_halls
```